MuxExoPlayer/src/main/java/com/mux/stats/sdk/muxstats/MuxStatsExoPlayer.kt

```diff
@@ -138,13 +137,17 @@ class MuxStatsExoPlayer @JvmOverloads constructor(
     // Init MuxStats (muxStats must be created last)
     MuxStats.setHostDevice(MuxDevice(context))
     MuxStats.setHostNetworkApi(network)
+    // Generate a player ID (for our instance-tracking) unless one was supplied
+    if (!::playerId.isInitialized) {
+      playerId = context.javaClass.canonicalName!! + (playerView?.id ?: "audio")
```

Comment:
what does this evaluate to, just out of curiosity?

Reply:
@jsanford8 Something like `"com.some.company.app.activity.PlayerActivity3522..."` or an int-sized number. If they use proguard it would be `"a.b.a.c.v.d25235899..."`.  It doesn't really matter what it is, just that it's (reasonably) unique to an activity and view.